packages/aws-appsync-subscription-link/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aws-appsync-subscription-link",
-  "version": "2.2.3",
+  "version": "2.2.4",
```

Comment:
I think this is not necessary, lerna will do it automatically

Reply:
Will remove!